packages/affiliates/apps/DeployerRewards.js

```diff
@@ -36,16 +33,22 @@ async function App(config) {
 
   const rewards = DeployerRewards({
     queries,
-    empCreatorAbi,
     empAbi,
     coingecko,
     synthPrices
   });
 
+  // API has changed, we need to validate input. Emps will be required to include payout address.
+  empWhitelist = empWhitelist.map(empInput => {
+    rewards.utils.validateEmpInput(empInput);
+    // convert to standard eth checksum address otherwise lookups through BQ or web3 will fail
+    return empInput.map(rewards.utils.toChecksumAddress);
+  });
+
   // get emp info
   const { collateralTokens, collateralTokenDecimals, syntheticTokenDecimals, syntheticTokens } = await Promise.reduce(
     empWhitelist,
-    async (result, address) => {
+    async (result, [address]) => {
```

Comment:
nit: since this is now dealing with two addresses, it might be clearer to change this variable name to `empAddress`.